build.gradle

```diff
@@ -778,3 +779,14 @@ task compdb(type: Copy) {
     from "$buildDir/${hostName}/compile_commands.json"
     into "$projectDir"
 }
+
+if (project.hasProperty("anotherDistro")) {
+    targetList.each { targetName ->
+        task "${targetName}CheckAbiCompatibility"(type: CompareDistributionSignatures) {
+            dependsOn "${targetName}PlatformLibs"
+
+            target = targetName
+            oldDistribution = project.findProperty("anotherDistro")
```

Comment:
Should we add `onMismatchMode = FAIL` here? As soon as these tasks are not run regulary they are not going to fail on annotations making noise. If they are intended to be used as tasks to be run during the release, I think it's better if they fail to make release manager pay attention to them.

Reply:
Makes sense, thanks!